secp256k1-sys/wasm-sysroot/stdio.h

```diff
@@ -1,17 +0,0 @@
-#include <stddef.h>
```

Comment:
We can delete the entire file I guess?

Reply:
I don't know why these files are there, `stdlib.h` is empty as well.

Reply:
AFAIK @TheBlueMatt added them when he worked on wasm compatibility. They are included in the build-script but serve otherwise no purpose.

This may be an instance of Chesterton’s fence but I would honestly just try and delete them and if nothing breaks, you are good to go :D

Reply:
I tried removing it and build fails, FTR here is a portion of the error:
```
cargo:warning=In file included from depend/secp256k1/src/field.h:25:
cargo:warning=depend/secp256k1/src/util.h:16:10: fatal error: 'stdio.h' file not found
cargo:warning=#include <stdio.h>
cargo:warning=         ^~~~~~~~~
cargo:warning=1 error generated.
```

Reply:
Ah interesting. Yeah that makes sense!

Reply:
Yea, if nothing breaks these files (or any parts of them you can) should definitely be removed!

Reply:
Lol, I am pretty sure that `#include` should not be present upstream. cc @real-or-random

Reply:
I've created https://github.com/bitcoin-core/secp256k1/issues/1095 to track this.